anomalous-gas-js/src/agent.js

```diff
@@ -1,4 +1,3 @@
-const ethers = require("ethers");
```

Comment:
The code in these examples was never fully linted.  Eslint currently shows 269 issues.  After running with the --fix option,
here is what's left:

```
/home/ubuntu/forta-uma-suite/src/examples/anomalous-gas-js/src/agent.js
  3:29  error  Unable to resolve path to module 'rolling-math'                          import/no-unresolved
  8:35  error  'rollingMath' is already declared in the upper scope on line 6 column 7  no-shadow

/home/ubuntu/forta-uma-suite/src/examples/anomalous-value-js/src/agent.js
   9:39  error  A constructor name should not start with a lowercase letter                    new-cap
  11:35  error  'contractAddresses' is already declared in the upper scope on line 8 column 7  no-shadow
  11:54  error  'provider' is already declared in the upper scope on line 9 column 7           no-shadow
  43:51  error  Expected '!==' and instead saw '!='                                            eqeqeq
  45:7   error  Assignment to property of function parameter 'contractAddresses'               no-param-reassign

/home/ubuntu/forta-uma-suite/src/examples/anomalous-value-js/src/agent.spec.js
   3:7   error  'RollingMath' is assigned a value but never used             no-unused-vars
  14:39  error  A constructor name should not start with a lowercase letter  new-cap

/home/ubuntu/forta-uma-suite/src/examples/big-tx-js/src/agent.js
  2:7  error  'BigNumber' is assigned a value but never used  no-unused-vars

/home/ubuntu/forta-uma-suite/src/examples/big-tx-js/src/agent.spec.js
  11:1  error  This line has a length of 109. Maximum allowed is 100  max-len

/home/ubuntu/forta-uma-suite/src/examples/deployment-blacklist-js/src/agent.js
  1:7   error  'ethers' is assigned a value but never used         no-unused-vars
  2:7   error  'BigNumber' is assigned a value but never used      no-unused-vars
  4:42  error  'getJsonRpcUrl' is assigned a value but never used  no-unused-vars

/home/ubuntu/forta-uma-suite/src/examples/deployment-blacklist-js/src/agent.spec.js
  14:1  error  This line has a length of 167. Maximum allowed is 100  max-len

/home/ubuntu/forta-uma-suite/src/examples/function-call-js/src/agent.spec.js
  68:9   error  'to' is assigned a value but never used    no-unused-vars
  68:25  error  'data' is assigned a value but never used  no-unused-vars

/home/ubuntu/forta-uma-suite/src/examples/transfer-ownership/src/agent.js
  15:27  error  Expected '!==' and instead saw '!='  eqeqeq
```